src/main/webapp/js/common.js

```diff
@@ -587,6 +587,38 @@ function setStatusMessage(message, status) {
 }
 
 /**
+ * Sets a status message and the message status to a given form.
+ * Default message type is info.
+ *
+ * @param message the text message to be shown to the user
+ * @param status type
+ * @param form form which should own the status
+ */
+function setStatusMessageToForm(message, status, form) {
+    if (message === '' || message === undefined || message === null) {
+        return;
+    }
+
+    var $statusMessagesToUser = $(DIV_STATUS_MESSAGE);
+    var $statusMessage = $('<div></div>');
+    
+    $statusMessage.addClass('overflow-auto');
+    $statusMessage.addClass('alert');
+    // Default the status type to info if any invalid status is passed in
+    $statusMessage.addClass('alert-' + (StatusType.isValidType(status) ? status : StatusType.INFO));
+    $statusMessage.addClass('statusMessage');
+    $statusMessage.html(message);
+    
+    $statusMessagesToUser.empty();
+    $statusMessagesToUser.append($statusMessage);
+    
+    // Copy the statusMessage and prepend to form
+    var $copyOfStatusMessagesToUser = $statusMessagesToUser.clone().show();
+    $statusMessagesToUser.remove();
+    $(form).prepend($copyOfStatusMessagesToUser);
+}
+
+/**
```

Comment:
extracted some parts to a common function. Since it is written in a generic way, I guess keeping it in common.js `would` make more sense. For other pages, if needed (show the status close to the form), we can reuse this.